planner/core/explain.go

```diff
@@ -161,7 +161,8 @@ func (p *PhysicalSelection) ExplainInfo() string {
 
 // ExplainInfo implements PhysicalPlan interface.
 func (p *PhysicalProjection) ExplainInfo() string {
-	return string(expression.ExplainExpressionList(p.Exprs))
+	return string(expression.ExplainExpressionList(p.Exprs)) +
+		fmt.Sprintf(", ProjectionConcurrency:%d", p.ProjectionConcurrency)
```

Comment:
How about this, it's shorter:
```suggestion
		fmt.Sprintf(", concurrency:%d", p.ProjectionConcurrency)
```

BTW, if concurrency is `0`, it's no need to print it out.